test/util/lib/LibTestConstants.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: CAL
+pragma solidity ^0.8.18;
+
+import "openzeppelin-contracts/contracts/utils/introspection/IERC1820Registry.sol";
```

Comment:
this is here https://github.com/rainlanguage/rain.erc1820/blob/main/src/interface/IERC1820Registry.sol#L23